engine/lbge.asd

```diff
@@ -7,7 +7,8 @@
     ((:file "hash-map")))
    (:module image-loader
     :components
-    ((:file "packages")
+    ((:file "image")
```

Comment:
`packages` should always be the first file.
There is no way to forward-define a package in CL, and asdf loads files sequentially.
So if you have `use-package` in some file, which refers to package which`defpackage` has not yet been encountered, you will receive a error, what just happened to your travis build.